index.html

```diff
@@ -1,19 +1,127 @@
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+  <!-- Body Starts Here -->
+  <body>
+    <!-- Header Starts Here -->
+    <header class="header">
+      <img
+        src="img/karma-logo.svg"
+        width="100%"
+        alt="karma-logo"
+        class="navbar-logo"
+      />
+      <nav>
+        <button>
+          <a href="#"><strong>Meet Karma</strong></a>
+        </button>
+        <button><a href="#">How it Works</a></button>
+        <button><a href="index.html">Store</a></button>
+        <button><a href="#">Blog</a></button>
+        <button><a href="#">Help</a></button>
+        <button><a href="#">Login</a></button>
+      </nav>
+    </header>
 
-</body>
+    
+    <!-- Main Starts Here -->
+    <main class="main">
+      <!-- Article One Starts Here -->
+      <div class="first-section">
+        <h1 style="font-size: 4vw">Introducing Karma</h1>
+        <h3 style="font-size: 2vw">Bring WiFi with you, everywhere you go.</h3>
+        <button class="button-orange">Learn More</button>
+      </div>
+
+      <!-- Article Two Starts Here -->
+      <h1 style="font-size: 3vw">
+        <center>Everyone needs a little Karma.</center>
+      </h1>
+
+      <ol style="list-style: none" class="second-section">
+        <li class="main-links">
+          <img
+            src="img/icon-devices.svg"
+            alt="icon-devices"
+            class="img-resize"
+          />
+          <a href="#" style="font-size: 2vw">Internet for all Devices</a>
+        </li>
+        <li class="main-links">
+          <img src="img/icon-coffee.svg" alt="icon-coffee" class="img-resize" />
+          <a href="#" style="font-size: 2vw">Boost Your Productivity</a>
+        </li>
+        <li class="main-links">
+          <img src="img/icon-refill.svg" alt="icon-refill" class="img-resize" />
+          <a href="#" style="font-size: 2vw">Pay as You Go</a>
+        </li>
+      </ol>
+
+      <!-- Article Three Starts Here -->
+      <div class="third-section">
+        <img
+          src="level-2/homepage-feature.png"
+          alt="homepage-feature"
+          class="homepage-feature"
+        />
+        <div class="testimonial">
+          <h3 style="font-size: 3vw">
+            "Whatever I am, I just don't worry about my connection anymore!"
+          </h3>
+          <button class="button-orange"><a href="store.html">Get Karma today<a></button>
+        </div>
+      </div>
+    </main>
+<hr />
+
+    <!-- Footer One Starts Here -->
```

Comment:
I think there may be something wrong with this. It seems to have generated a column of a tag on the right.
<img width="1227" alt="Screenshot 2022-11-12 at 14 11 45" src="https://user-images.githubusercontent.com/40305104/201475576-45c0e345-5813-4619-a2e4-ca0426fd03a8.png">